backend/Squirrel.ConsoleApp.Models/Models/DbEngine.cs

```diff
@@ -2,6 +2,6 @@
 
 public enum DbEngine
 {
-    SqlServer,
+    MsSqlServer,
```

Comment:
not sure that we should rename it - `SqlServer` is a common name

Reply:
I saw it here: [enum](https://github.com/BinaryStudioAcademy/bsa-2023-squirrel/blob/development/backend/Squirrel.Core/Squirrel.Core.DAL/Enums/DbEngine.cs)

Reply:
Ok, I will do it